kerl

```diff
@@ -733,6 +743,63 @@ do_deploy()
     echo "kerl_deactivate"
 }
 
+is_valid_install_path()
+{
+    # don't allow installs into home directory
+    if [ "$1" = "$HOME" ]; then
+        echo "ERROR: You cannot install a build into $HOME. It's a really bad idea."
+        return 1
+    fi
+
+    # don't allow installs into .erlang because
+    # it's a special configuration file location
+    # for OTP
+    if [ "$1" = "$HOME/.erlang" ]; then
+        echo "ERROR: You cannot install a build into $HOME/.erlang. (It's a special configuration file location for OTP.)"
+        return 1
+    fi
+
+    # don't install into .kerl either.
+    if [ "$1" = "$HOME/.kerl" ]; then
```

Comment:
Perhaps we can use `readlink -f $1` to compare? Although that might cause issues if `$HOME` lives under a symlink... So we'd probably need to check `if [ "$(readlink -f "$1")" = "$(readlink -f "$HOME/.kerl")" ]`...

Complicated but it should work...

NB We don't want to use `readlink -f ...` to try to sanitise the input beforehand, because it could cause surprises with the resultant install location store in `otp_installations`, I think